lib/sparql_to_sw_solr/physical_field.rb

```diff
@@ -0,0 +1,51 @@
+module SparqlToSwSolr
+  class InstanceSolrDoc
+    module PhysicalField
+
+      private
+
+      def physical_values
+        physical_solutions.each_solution do |soln|
+          next unless soln.bindings.keys.include?(:noteType)
+          @note_type = soln.noteType.to_s
+          @physical1 = soln.extentLabel_1.to_s
+          @physical3 = soln.instanceDimensions_3.to_s
+          note_case(soln)
```

Comment:
so the way this is written, I think we will get only the last value if there are multiple values.   In theory, there can be multiple values, so it should add a result to an array for each soln.  (In practice this will rarely, if ever, happen ...)